### DISCUSSION TITLE
How to customize the Klipperscreen

### DISCUSSION THREAD

**Discussion by ReyhaneVahidi** [KlipperScreen/KlipperScreen#1337] [General]
Hi, I need to customize the Klipperscreen for a project, I want to remove some items from the UI because we generally just to move a stepper motor forward and backward and we don't need the items like temprature. I wanted to know would it be possible to remove some items?
Thanks for the help.

**Comment by alfrix** [comment]
KlipperScreen adapts to the things configured in Klipper, so you shouldn't need to do modify anything for your use case.
